### DISCUSSION TITLE
Welcome to Smart-Home Discussions!

### DISCUSSION THREAD

**Discussion by ajumalp** [ajumalp/Smart-Home#3] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by hokkar** [comment]
Hi there,

If you are looking for ways to integrate Moes smart devices into this setup, I found a helpful discussion/guide here: https://moessmartb2b.com/

Usually, these devices work great via Zigbee2MQTT / ZHA. Hope this helps anyone looking for the same solution!
